kimchi/src/verifier.rs

```diff
@@ -886,11 +889,15 @@ where
                 mds: &G::sponge_params().mds,
                 zk_rows,
             };
-            let challenges = Challenges {
+            let challenges = BerkeleyChallenges {
                 alpha: oracles.alpha,
                 beta: oracles.beta,
                 gamma: oracles.gamma,
-                joint_combiner: oracles.joint_combiner.as_ref().map(|j| j.1),
+                joint_combiner: oracles
+                    .joint_combiner
+                    .as_ref()
+                    .map(|j| j.1)
+                    .unwrap_or(G::ScalarField::zero()),
```

Comment:
Even though it seems ok at a first sight, I do not remember if we rely on the fact the joint combiner is None somewhere.